setup.py

```diff
@@ -45,7 +45,7 @@
         "elements": [
             "element-animal>=0.1.8",
             "element-event>=0.2.3",
-            "element-interface>=0.6.0",
+            "element-interface>=0.7.0",
```

Comment:
Can we point this to the `multi-plane-processing` branch? I'm not sure if that's possible within `setup.py`